ViAn/Video/zoomer.h

```diff
@@ -63,6 +64,8 @@ class Zoomer{
     cv::Rect get_frame_rect() const;
     cv::Rect get_view_rect() const;
 
+    void enforce_frame_boundaries();
```

Comment:
Small thing but i would put this above the setters and getters since it doesnt belong with them